mappings/net/minecraft/entity/Entity.mapping

```diff
@@ -7,6 +7,7 @@ CLASS net/minecraft/class_1297 net/minecraft/entity/Entity
 	FIELD field_22467 pos Lnet/minecraft/class_243;
 	FIELD field_22468 blockPos Lnet/minecraft/class_2338;
 	FIELD field_23807 inanimate Z
+	FIELD field_25154 chunkPosUpdateNeeded Z
```

Comment:
Should the field name match the function here?

Reply:
Yeah would be better to match